src/main/java/roomescape/reservation/domain/Reservation.java

```diff
@@ -75,6 +77,10 @@ public ReservationStatus getStatus() {
         return status;
     }
 
+    public Long getAMOUNT() {
```

Comment:
메서드 이름은 카멜케이스로 작성해주셔도 됩니다 :)

```suggestion
    public Long getAmount() {
```